translation-sh/conf/mapper.tmp

```diff
@@ -0,0 +1,12 @@
+#######################################
+# The Mapper Template
+# 1. Copy file to conf/.mapper
+# 2. Update values if needed
+# 3. Use tab to separate the columns
+#######################################
+# VERSION  FILENAME   FOLDER
+10	translationlist_1	details1
+15	translationlist_1_5	details1_5
+25	translationlist	details
```

Comment:
```suggestion
25	translationlist		details
```

Reply:
This just a template, and it actually has a tab, but the string length makes it look like a space, but it is in fact a tab character ;)